lstchain/image/muon/muon_analysis.py

```diff
@@ -56,7 +49,7 @@ def pixel_coords_to_telescope(geom, equivalent_focal_length):
     )
     tel_coord = camera_coord.transform_to(TelescopeFrame())
 
-    return tel_coord.delta_az, tel_coord.delta_alt
```

Comment:
`delta_alt` and `delta_az` should also be modified in the function docstring

Reply:
Done